test/services/database-analyzer/database-analyzer-sequelize.js

```diff
@@ -17,7 +18,7 @@ describe('Database analyser > Sequelize', () => {
 
   function performDatabaseAnalysis(connection, dialect) {
     const databaseAnalyzer = new DatabaseAnalyzer(connection, { dbDialect: dialect });
-    return databaseAnalyzer.perform();
+    return databaseAnalyzer.perform('customers');
```

Comment:
Since this PR only adds the function `TableForeignKeysAnalyzer` and tests it, we should not have this change, isn't it?